artio-core/src/main/java/uk/co/real_logic/artio/library/SessionConfiguration.java

```diff
@@ -238,6 +250,21 @@ public Builder address(final String host, final int port)
             return this;
         }
 
+        public Builder commonHeaderLength(final int commonHeaderLength)
```

Comment:
Is it safe to allow people to set the commonHeaderLength independently of the useFixTTransport property?